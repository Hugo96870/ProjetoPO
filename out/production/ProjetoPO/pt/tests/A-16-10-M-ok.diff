*** tests/A-16-10-M-ok.out	2020-12-01 16:55:11.000000000 +0000
--- tests/A-16-10-M-ok.outhyp	2020-12-03 15:05:33.530832800 +0000
***************
*** 16,22 ****
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: Produto '6M2': pedido=10000, existências=100
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
--- 16,22 ----
  3 - Registar encomenda
  4 - Pagar
  0 - Sair
! Escolha uma opção: Identificador do cliente: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: Produto '6M2': pedido=0, existências=0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar venda
