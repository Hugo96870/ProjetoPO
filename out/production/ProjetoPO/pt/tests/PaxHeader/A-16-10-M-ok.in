30 mtime=1606841711.235640734
