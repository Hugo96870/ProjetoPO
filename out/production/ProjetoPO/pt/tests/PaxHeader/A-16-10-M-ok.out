30 mtime=1606841711.236403302
